### ISSUE TITLE
Support for HTTP/2 and HTTP/3 based on 3GPP upgrades

### ISSUE THREAD

**Issue by haudiobe** [Dash-Industry-Forum/Ingest#162]
In 3GPP a draft specification is defned to add HTTP/3 to the specification. In the context of this, it was checked if DASH-IF Ingest Protocol would be upgraded to HTTP/3. Below some text and the note. It would be good to get some feedback.

8.3	DASH-IF push-based content ingest protocol
If IngestConfiguration.protocol is set to urn:3gpp:5gms:content-protocol:dash-if-ingest in the Content Hosting Configuration, media resources shall be ingested by the 5GMSd AS, as specified by the DASH IF Live Media Ingest specification [3]. The IngestConfiguration.pull property shall be set to False, indicating that a Push-based protocol is used. The IngestConfiguration.baseURL property shall be set by the 5GMSd AF to the base URL that is to be used by the 5GMSd Application Provider to upload the DASH segments and MPD(s) to the 5GMSd AS at reference point M2d.

_NOTE :	The ingest protocol in [3] is defined for HTTP/1.1 [24] only. If that specification is updated to support HTTP/2 [31] and/or HTTP/3 [HTTP/3], these HTTP versions may also be used for push-based content ingest via reference point M2d._

**Comment by RufaelDev** [comment]
@haudiobe good point to take into account for the next revision, one thing to think about is that the DASH-IF ingest spec is not necessarily considering the mobile device scenario but mostly cloud encoder/packager. But the extension to mobile scenario in 3GPP is logical and so is using new protocols, but it may require some other considerations that have not been considered in DASH-IF.
